.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "submodules/island.is"]
+	path = submodules/island.is
+	url = https://github.com/island-is/island.is
```

Comment:
Do you need to submodule the whole island.is repo? It would be nice if you can somehow just submodule/subtree the `libs/island-ui` folder.

Reply:
The submodule is versioned to some tag or commit hash, right?

Reply:
Not an git expert but looking the .gitmodules file, it looks like I can add an `branch = main` to tell the submodule to follow that branch, then we can run `git submodule --update` to get the latest changes.

Reply:
There is a technique with `sparse-checkout` that I am not familiar with. Maybe some from team andes could solve this for us, that would be great.